requirements.txt

```diff
@@ -1,6 +0,0 @@
-click
-daiquiri
-pipenv
-requests
-toml
-delegator.py
```

Comment:
The requirements file is needed. One can install Kebechet from PyPI - in that case requrements.txt is used.